superdesk/text_checkers/ai/imatrics.py

```diff
@@ -149,13 +152,18 @@ def analyze(self, item: dict) -> dict:
             "pubStatus": False,
             "headline": headline,
             "body": body,
+            "language": item["language"],
```

Comment:
are we sure that `language` is always set here?

Reply:
it is https://github.com/superdesk/superdesk-core/blob/develop/apps/archive/common.py#L190-L191